third_party/test/proto/WORKSPACE

```diff
@@ -0,0 +1,43 @@
+workspace(name = "proto")
```

Comment:
Is it the first place we hardcode a WORKSACE in one of the tests? Maybe we better use one generator to ease on us? Not saying that it's mandatory for this PR but can be nice to open an issue and care for it later.

I know that using[ `bazel-integration-testing` ](https://github.com/bazelbuild/bazel-integration-testing)can be challenging for something larger then a rule that is contained in single bzl file but I wish we had a better solution to do proper e2e tests on this rather than this.